bench/run.js

```diff
@@ -31,31 +28,31 @@ init.then(() => {
   let sourcemapBuffer = sourcemapInstance.toBuffer();
   let rawSourceMap = sourcemapInstance.toVLQ();
 
-  suite.add("consume vlq mappings", async () => {
+  suite.add('consume vlq mappings', async () => {
     let map = new SourceMap();
-    map.addRawMappings(
-      rawSourceMap.mappings,
-      rawSourceMap.sources,
-      rawSourceMap.names
-    );
+    map.addRawMappings({
```

Comment:
Is that a breaking API change? 
If you do `yarn add parcel@next` at the moment, it will actually install `@parcel/source-map@2.0.0-alpha.4.11`

Reply:
It unfortunately is, it was this or an additional function argument, both seem equally bad solution with this one being slightly more future proof in case we'd need to support more map fields

Reply:
We could merge this and just not release a source-map update until we release a new alpha?